<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: black;

}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>320</width>
      <height>230</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="2">
      <widget class="QPushButton" name="EQUALS">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #006400;
 padding: 5px;

}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="MODULO">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #008000;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="DZIEWIEC">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="MEM_GET">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #228B22;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>M_GET</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="CZTERY">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="MEM_SUB">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #228B22;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>M-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="MULTIPLY">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #008000;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="SZESC">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="ZERO">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="TRZY">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="MEM_ADD">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #228B22;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>M+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="DWA">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="SIEDEM">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="BACKSPACE">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #008000;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>BACKSPACE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="PIEC">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="MEMORY_CLEAR">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #228B22;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>M_CLEAR</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="SUB">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #008000;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="OFF">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #008000;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>OFF</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="CLEAR">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #228B22;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>CLEAR</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="DOT">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #228B22;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="DIVIDE">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #008000;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="OSIEM">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="ADD">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color:  #008000;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="JEDEN">
       <property name="styleSheet">
        <string notr="true">QPushButton{
     background-color: #696969;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>TRZY</zorder>
    <zorder>DWA</zorder>
    <zorder>PIEC</zorder>
    <zorder>SZESC</zorder>
    <zorder>MEMORY_CLEAR</zorder>
    <zorder>SIEDEM</zorder>
    <zorder>CZTERY</zorder>
    <zorder>OSIEM</zorder>
    <zorder>DIVIDE</zorder>
    <zorder>MODULO</zorder>
    <zorder>DZIEWIEC</zorder>
    <zorder>JEDEN</zorder>
    <zorder>CLEAR</zorder>
    <zorder>MULTIPLY</zorder>
    <zorder>SUB</zorder>
    <zorder>MEM_ADD</zorder>
    <zorder>MEM_SUB</zorder>
    <zorder>MEM_GET</zorder>
    <zorder>OFF</zorder>
    <zorder>BACKSPACE</zorder>
    <zorder>ADD</zorder>
    <zorder>EQUALS</zorder>
    <zorder>DOT</zorder>
    <zorder>ZERO</zorder>
   </widget>
   <widget class="QLineEdit" name="EKRAN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
     background-color: black;
     border: 1px solid  #006400 ;
color: #006400;
}    
</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="EKRAN2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 14pt &quot;Segoe UI&quot;;
     background-color: black;
     border: 1px solid  #006400 ;
color: #006400;
}    
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="EKRAN3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 14pt &quot;Segoe UI&quot;;
     background-color: black;
     border: 0px;
 color: #006400;
}    
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="KONWERSJA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
     background-color:  #008000;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
    </property>
    <property name="text">
     <string>CONVERT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="z_jakiego">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
     background-color:  #228B22;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
    </property>
    <item>
     <property name="text">
      <string>Binarny</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ósemkowy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dziesiętny</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Szesnastkowy</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="na_jaki">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
     background-color:  #228B22;
 padding: 5px;
}    
QPushButton:pressed{
     background-color: #A9A9A9;
 padding: 5px;
}    </string>
    </property>
    <item>
     <property name="text">
      <string>Binarny</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ósemkowy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dziesiętny</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Szesnastkowy</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Matrix">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
     background-color: black;
     border: 0px;
}    
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMENU">
    <property name="title">
     <string>MENU</string>
    </property>
    <addaction name="actionAUTOR"/>
   </widget>
   <addaction name="menuMENU"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAUTOR">
   <property name="text">
    <string>AUTOR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
